### ISSUE TITLE
Authorship order is not retained, forced to alphabetical

### ISSUE THREAD

**Issue by Noxxaa** [adelikat/tasvideos#431]
Example:

http://tasvideos.org/4505M.html author: "PracticalTAS, ajp_anton, tauKhan & Mario 64 Master"

https://tasvideos.mistflux.net/4505M author: "Mario 64 Master, PracticalTAS, ajp_anton, taukhan"

The old site allows the list of authors to be arranged in any order. The new site forces alphabetical order for users registered on the site, followed by alphabetical order for users not registered on the site.

I think this is a significant regression in terms of clarity surrounding authorship, as it gives authors/teams less control as to the ordering of their names. Normally, the primary author of a movie is assumed to be the first name in the list (as that's the most visible, by nature of being read first). Now, there is no guarantee that that is the case. This can cause a lot of confusion as to which author was the main contributor to a published TAS, and makes correct attribution a lot more difficult.

While some people do not care about order of authorship, I'm certain that many people do, and forcing their names to a set order that doesn't represent the work that the respective authors put in, is tantamount to incorrect author attribution. This is a major issue, in my opinion.

**Comment by Masterjun3** [comment]
Non-alphabetical order *is* possible on the current site when creating a new submission. Maybe the database import accidentally sorts them.
However, it's true that registered and non-registered users can't be mixed in the order, so the points still apply there.

**Comment by adelikat** [comment]
The correct design here, is an Ordinal column on both SubmissionAuthors and PublicationAuthors tables.  A PR must be based on this design.
